services/motion/builtin/builtin_test.go

```diff
@@ -430,6 +431,7 @@ func TestMoveOnMapLongDistance(t *testing.T) {
 	t.Run("test tp-space planning on office map", func(t *testing.T) {
 		t.Parallel()
 		ms := createMoveOnMapEnvironment(ctx, t, "slam/example_cartographer_outputs/viam-office-02-22-3/pointcloud/pointcloud_4.pcd")
+		defer ms.Close(context.Background())
```

Comment:
All these motionService.Close calls need to be added so that tests don't leak go routines.